index.js

```diff
@@ -45,21 +56,18 @@ function isCurrentCityDataLoaded() {
   return cityByWeather.hasOwnProperty(currentCity);
 }
 
-renderApp();
-
 function reRenderApp() {
   // Need to split rendering cycles
-  setTimeout(() => window.renderApp(), 0);
+  setTimeout(window.renderApp, 0);
```

Comment:
Why is that required?

A side note: if there is a delay assigned anyway, perhaps we could use a variable delay to emulate network requests that take longer when re-render gets fired on actual data load from remote server.

Reply:
Regarding splitting rendering - see [this thread ](https://github.com/kottans/framework-2021/pull/20#discussion_r607600328)

We could use dev tools to emulate network delays etc. Don't want to make it overcomplicated.